src/Response/Product/GetResponse.php

```diff
@@ -99,7 +99,7 @@ public function hasProductData(): bool
     /**
      * @return int|null
```

Comment:
Aj moje chyba, ale oprav to i v komentáři než to mergnu pls

Reply:
@romanpetras-expando Upraveno